src/gc.c

```diff
@@ -135,6 +135,11 @@ typedef struct _bigval_t {
         size_t sz;
         uintptr_t age : 2;
     };
+#ifdef _P64
+    void *_padding[8 - 4];
+#else
+    void *_padding[16 - 4];
+#endif
```

Comment:
Took this from [here](https://github.com/JuliaLang/julia/commit/e60a64d75aa1c51e0bbb465fdf227f87f133e1a7#diff-31b93f4272eaeede3738531e90317e0cR105), it's necessary for ensuring that GC header doesn't accidentally offset `data` from 64-byte alignment.  Is there any value to keeping these values `8 - 4` and `16 - 4`, or should I replace them with `4` and `12` respectively?